src/ProcessManager.js

```diff
@@ -33,6 +35,20 @@ class ProcessManager extends EventEmitter {
     this.emit('killed-process', pid, this.window);
   }
 
+  DebugProcess(pid) {
```

Comment:
Conventionally the case is `debugProcess`

Reply:
done